tests/test_compute.py

```diff
@@ -15,13 +15,31 @@
 from tests.test_helpers import (
     get_consumer_wallet,
     get_publisher_wallet,
-    get_dataset_ddo_with_compute_service_no_rawalgo, get_dataset_ddo_with_compute_service_specific_algo_dids, get_algorithm_ddo,
-    get_dataset_ddo_with_compute_service, get_compute_job_info, get_possible_compute_job_status_text, mint_tokens_and_wait, get_nonce,
+    get_dataset_ddo_with_compute_service_no_rawalgo, get_dataset_ddo_with_compute_service_specific_algo_dids,
```

Comment:
did you run isort on these imports? They do not seem to be ordered alphabetically, and actually it seems you haven't imported anything new, so no need to change the imports section.